classroom.rb

```diff
@@ -14,24 +14,36 @@ def add_student
 end
 
 class Book
+  @@books = []
   def initialize(title, author)
     @title = title
     @author = author
+    @@books.push({ 'title' => @title, 'author' => @author })
   end
   attr_accessor :title, :author
 
   def add_rental(date)
     new.Rental(date)
   end
+
+  def list_of_books
+    @@books.each_with_index do |book, idx|
+      puts "#{idx}) Title: #{book['title']}, Author: #{book['author']}"
+    end
+  end
+
+  def books_array
+    @@books
+  end
 end
 
 class Rental
-  def initialize(date, person_obj, book_obj)
+  @@rentals = []
```

Comment:
- [ ] Comment in the Book class applies here as well. You should move it to the App class instead. In the Rental class, we should only care about a single object and not a list of objects.